### PR TITLE
 Add option to toggle off inversion

### PR DESCRIPTION
**Pull request by sssangha**
Add option to toggle inversion in the config function. By default set to true, like for unwrapping.

This allows for more flexibility when running especially large-scale processing.

### CODE REVIEW

File: src/dolphin/workflows/displacement.py [hunk 1]
```diff
@@ -234,8 +234,7 @@ def run(
     # ##############################################
 
     ts_opts = cfg.timeseries_options
-    # Skip if we only have 1 unwrapped, or if we didn't ask for inversion/velocity
-    if len(unwrapped_paths) > 1 and (ts_opts.run_inversion or ts_opts.run_velocity):
+    if not ts_opts.run_inversion:
```
Comment (scottstanie) [inline src/dolphin/workflows/displacement.py:237]: Could you switch back these changes? I think this happened from me merging the other fixes in the last PR, which you may not have had in your branch

**Comment by scottstanie** [comment]
Sounds good, thanks! 
I've been thinking about how we can make the `config` CLI more automatic, so that as we add/change the pydantic classes, we don't need to keep updating the script to mirror it. But for now this works

File: src/dolphin/workflows/displacement.py (hunk 1 above)
Comment (sssangha) [reply src/dolphin/workflows/displacement.py]: Ok yeah I see this has already been addressed. I've adjusted this commit accordingly.

**Comment by sssangha** [comment]
> Sounds good, thanks! I've been thinking about how we can make the `config` CLI more automatic, so that as we add/change the pydantic classes, we don't need to keep updating the script to mirror it. But for now this works

Yeah that would be great! And glad to help.

**Pull request merged**